File: cpp/bc1133.cpp
#include <iostream>

using namespace std;

int main()
{
	int fVal, sVal, aux;
	
	cin >> fVal >> sVal;
	
	if(fVal > sVal)
	{
		aux = sVal;
		sVal = fVal;
		fVal = aux;
	}
	
	for(int i = fVal+1; i < sVal; i++)
		if(i % 5 == 2 || i % 5 == 3) cout << i << endl;
}

File: python/bc1133.py
fVal = int(input())
sVal = int(input())
if fVal > sVal:
    aux = sVal
    sVal = fVal
    fVal = aux

for c in range(fVal+1, sVal):
    if c % 5 == 2 or c % 5 == 3: print(c)